common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4607,6 +4614,12 @@
     ],
     "sqlState" : "42K03"
   },
+  "PERSISTED_VIEW_READS_FROM_TEMPORARY_VIEW" : {
```

Comment:
how is it different from `INVALID_TEMP_OBJ_REFERENCE`?

Reply:
I suspect this is vestigial. I switched to `INVALID_TEMP_OBJ_REFERENCE` in the upstream PR.